cypress/e2e/hermioneAccount.cy.js

```diff
@@ -0,0 +1,91 @@
+import { faker } from '@faker-js/faker';
+import { utils } from '../support/utils.js';
```

Comment:
Ensure that the `utils` module is correctly exported from '../support/utils.js'. If the module or the `formatCurrentDate` function is not exported properly, it will cause an error when trying to use `utils.formatCurrentDate()` on line 61.